devito/ir/cgen/printer.py

```diff
@@ -401,6 +401,7 @@ def _print_Fallback(self, expr):
     _print_IndexSum = _print_Fallback
     _print_ReservedWord = _print_Fallback
     _print_Basic = _print_Fallback
+    _print_VectorAccess = _print_Fallback
```

Comment:
This made me think, is this `_print_Fallback` actually necessary, the printer always defaults to `str(expr)` if it can't find a print method but I guess it bypasses the whole "search for a print method"

Reply:
yes, it would not do the expected thing without it, but I don't know why